app/src/renderer/js/main.js

```diff
@@ -165,15 +165,17 @@ document.addEventListener('DOMContentLoaded', () => {
         windowTitle.innerText = 'Kap';
         time.innerText = '00:00';
         size.innerText = '0 kB';
+        const type = exportAs.value;
 
-        if (exportAs.value === 'mp4') {
+        restoreInputs();
+
+        if (type === 'mp4') {
```

Comment:
Would it be worth changing this to case/switch syntax? IMHO it's bit more readable.